### ISSUE TITLE
Keyboard shortcut Shift-Control-Option-Command-§ crashes Due forever, requiring a reinstall?

### ISSUE THREAD

**Issue by junjie** [dueapp/Due-macOS#97]
Got a strange bug report about the above:

> I found that one shortcut will crash due forever.
> The only solution is to reinstall Due.
> 
> Please have a look
> 
> Shift + Control + Option + Command and §

User is using en_PT (English (Portugal)). I asked about the § key, it is located here:

![JC ScreenShot 2020-08-31 at 16 34 59](https://user-images.githubusercontent.com/524420/92307668-9f1afe00-efca-11ea-925d-9a83682d4bbf.jpg)

I sent a followed up email enquiring:

> Thanks! To be clear, are you saying that if you use this keyboard shortcut in Due > Preferences > Show/Hide Window, it will crash Due forever?

Awaiting response.

**Comment by junjie** [comment]
> Yes, that is correct.

**Comment by honcheng** [comment]
@junjie how did you manage to set en_PT? My English options don't have PT

**Comment by junjie** [comment]
<img width="1275" alt="CleanShot 2020-09-11 at 11 48 28@2x" src="https://user-images.githubusercontent.com/524420/92855389-cb000e80-f424-11ea-8ba3-1199eae50a8a.png">

Try a Portuguese or Brazilian keyboard. I had to press down on Opt to get the key. Not easy to screencap that.

This virtual keyboard can be found here:

![CleanShot 2020-09-11 at 11 49 39@2x](https://user-images.githubusercontent.com/524420/92855474-e2d79280-f424-11ea-9028-c7fc381baf7d.png)

**Comment by honcheng** [comment]
![CleanShot 2020-09-11 at 11 56 03](https://user-images.githubusercontent.com/208827/92856514-1a930a00-f426-11ea-9343-942bcbe589d4.gif)

I mean I cannot bring up the double S symbol when I try Portuguese keyboard. Tried Brazilian too.

**Comment by honcheng** [comment]
Oh. It's somewhere else.
Anyway, I cannot get the symbol in the shortcut recorder using the screen keyboard. Will find another way.

**Comment by honcheng** [comment]
I can reproduce this using Apple Script 

```
delay 2
tell application "System Events"
	key code 10 using {shift down, command down, option down, control down}
end tell
```

I referenced the key code here https://apple.stackexchange.com/a/36947
§ is 10 

Managed to crash it permanently. The character isn't supported. 

I'll try to first fix the crash so it does not crash with unsupported characters,  then support the § character if possible.

**Comment by honcheng** [comment]
Done. Will be available in 20.6
